cmd/webhook/cmd.go

```diff
@@ -63,6 +65,8 @@ func New() *cobra.Command {
 
 func run() func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
+		const openshiftSecurityGVR = "security.openshift.io/v1"
```

Comment:
could you please move it outside of the function, to the top level where we usually declare constants?